### DISCUSSION TITLE
Create a GitHub Release when a Fluentd release is made

### DISCUSSION THREAD

**Discussion by stevehipwell** [fluent/fluentd#3625] [Ideas]
### Is your feature request related to a problem? Please describe.

The current release flow for Fluentd creates a tag in the format `vX.Y.Z` which works fine for direct use by Ruby but doesn't follow the GitHub idiom of creating a "GitHub Release". As the GitHub idiom isn't being followed the latest version isn't shown on the repo default view, you can't watch releases and getting the latest version via the API is more complicated than it could be.

### Describe the solution you'd like

I'd like a GitHub Actions workflow to be added that watches for the release tags to be pushed to the repo. This workflow can make the tag into a "GitHub Release" with all of the benefits that entails.

This workflow could also be used to automate the creation of an issue and potentially a PR in the [fluent/fluentd-docker-image](https://github.com/fluent/fluentd-docker-image) repo.

### Describe alternatives you've considered

n/a

### Additional context

I'll open a PR to do this and for the implementation to be discussed.

**Comment by patrick-stephens** [comment]
I've got a working example for cmetrics as well that also does a floating `unstable` release of `master` commits: https://github.com/calyptia/cmetrics/blob/b488fceb0390baab69306216705d21998a4ae980/.github/workflows/packages.yaml#L60-L66
We're looking at a similar workflow on fluent-bit shortly although in that case the workflow will create the tag and release (subject to test results).

**Comment by fujimotos** [comment]
FWIW, starting from Fluentd v1.14.5, I started to publish a relaese on GitHub
as well. Here is how it looks like:

https://github.com/fluent/fluentd/releases/tag/v1.14.5

The announcement format etc. is not fully fixed yet, though. Please adds comment in
this thread if there is any feedback/discussion.
